1_fetch/src/get_sb_data.R

```diff
@@ -241,4 +241,21 @@ calc_avg_monthly_weather <- function(sb_data) {
     select(COMID, label, avg_monthly) %>%
     pivot_wider(names_from = "label", values_from = "avg_monthly")
   return(weather)
+}
+
+calc_avg_wildfire <- function(sb_data) {
+  sb_data <- read_csv(sb_data, show_col_types = FALSE) %>%
+    suppressMessages()
+  wildfire <- sb_data %>%
+    select(COMID, contains("_WILDFIRE_")) %>%
+    pivot_longer(!COMID, names_to = "name", values_to = "value") %>%
+    mutate(unit = str_sub(name, 1, 3), 
+           year = str_sub(name, 14, 17)) %>%
```

Comment:
In this case, the wildfire headers are pretty simple, with the first three characters describing CAT/ACC/TOT, followed by an underscore, then WILDFIREYYYY with YYYY as the year. But for other landscape attributes, like land cover, there are multiple combinations of descriptor/underscore/year, sometimes with an additional suffix to describe further breakdowns of data. So the headers are predictable within a sub-set of attribute, but across attribute types there is no formal naming convention. I think you suggestion would be better if names were more standardized, but since not I think I'll stick with the original.